python/sglang/srt/server_args.py

```diff
@@ -169,10 +169,9 @@ def __post_init__(self):
             gpu_mem = get_amdgpu_memory_capacity()
         else:
             gpu_mem = get_nvgpu_memory_capacity()
+
         if gpu_mem < 25000:
-            self.chunked_prefill_size //= 4  # make it 2048
-            self.cuda_graph_max_bs = 4
-            logger.info("Automatically adjust --chunked-prefill-size for small GPUs.")
+            logger.warning("Your GPU has less than 25GB memory. You may want to set a smaller --chunked-prefill-size (e.g., 512) to improve performance.")
```

Comment:
add smaller `--cuda-graph-max-bs` (e.g., 4) maybe?

Reply:
We have seving many models with SGLang in GTX 4090, the `cuda-graph-max-bs` can be large depend on the model parameter size, quantization method and qps. So I delete this settings.